bin/check-redis-list-length.rb

```diff
@@ -63,19 +63,38 @@ class RedisListLengthCheck < Sensu::Plugin::Check::CLI
          description: 'Redis list KEY to check',
          required: true
 
+  option :threshold,
```

Comment:
Actually, we could get rid of the option altogether, and just change the behaviour depending on the value of warning and critical. If warning < critical, then the comparision will be 'above', and if warning > critical is will be 'below'. I can make a new patch for this.